grantnav/frontend/templates/components/grants_table_scripts.html

```diff
@@ -27,6 +27,7 @@
 const params = new URLSearchParams(window.location.search);
 const inputRows = params.get('rows');
 const inputColumns = params.getAll('widgetColumn');
+const isWidget = window.location.href.includes('widget');
```

Comment:
And if you search for "widget" in grantnav search :) probably need to use window.location.pathname ?